README.md

```diff
@@ -6,17 +6,14 @@
 [![Docker Pulls][docker-badge]][docker]
 [![CII Best Practices][cii-badge]][cii]
 
-:balloon: Welcome to Linkerd! :wave:
+** This repo is for the old, 1.x version of Linkerd. Linkerd development is now
+happening in the [linkerd2][l5d2] repo.**
```

Comment:
```suggestion
** This repo is for the 1.x version of Linkerd.  
Most of the development efforts moved to the Kubernetes focused [linkerd2][l5d2] repo. **
```

Reply:
@peterfroehlich removing `old` looks good to me.

I'd like to be careful about a term like 'Most of` because Linkerd 1 has no new development (features, etc.) going on.

> Linkerd development efforts are happening in the [linkerd2](https://github.com/linkerd/linkerd2) repo, which is Kubernetes-focused